src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -81,7 +81,7 @@ object DockerPlugin extends AutoPlugin {
     ),
     dockerUpdateLatest := false,
     dockerEntrypoint := Seq(s"${(defaultLinuxInstallLocation in Docker).value}/bin/${executableScriptName.value}"),
-    dockerCmd := Seq(),
+    dockerCmd := (dockerCmd in Docker).value,
```

Comment:
This change does not work as `dockerCmd in Docker` is undefined. You can see the
error in the travis log outputs

```
docker:dockerCmd from *:dockerCmd ((com.typesafe.sbt.packager.docker.DockerPlugin) DockerPlugin.scala:84)
```